src/app/move-popup/sockets.html

```diff
@@ -4,12 +4,19 @@
     <div class="item-socket"
       ng-class="{ disabled: !socketInfo.enabled }"
       ng-repeat="socketInfo in category.sockets track by $index">
-      <img class="item-mod"
-        ng-src="{{ socket.displayProperties.icon | bungieIcon }}"
-        ng-class="{ notChosen: socket != socketInfo.plug }"
-        ng-repeat="socket in socketInfo.plugOptions track by $index"
-        press-tip="{{ socket.displayProperties.description }}{{ socketInfo.enableFailReasons }}"
-        press-tip-title="{{ socket.displayProperties.name }}">
+      <div class="socket-container"
+           ng-repeat="socket in socketInfo.plugOptions track by $index">
+        <div ng-if="socket != socketInfo.plug && socket.bestRated" class="circle-container">
+          <div class="blue-circle">
+            <i class="fa fa-circle" aria-hidden="true"></i>
```

Comment:
There's no need to use a fa-icon here. Just make a div that's the right size and has `border-radius: 50%`.